tools/scanner/scan.mk

```diff
@@ -40,8 +41,7 @@ scanner/image: output/docker_mlperf_scanner.stamp
 .PHONY: scanner/build
 scanner/build:
 	bazel clean
-	build-wrapper-linux-x86-64 --out-dir "${SONAR_OUT_DIR}" \
-		make flutter/android
+	build-wrapper-linux-x86-64 --out-dir "${SONAR_OUT_DIR}" make
```

Comment:
`flutter/prepare` target which is implied here takes some time. I think it would be better to revert it to `make flutter/android` until we enable Flutter code analysis.

Reply:
Done